plugins/rollbar-backend/src/api/types.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Make this re-usable with backend
+
+export type RollbarProjectAccessTokenScope = 'read' | 'write';
```

Comment:
This file is currently duplicated in the frontend. Other then making a `@backstage/plugin-rollbar-common` I can't think of a great way of sharing these types.

Reply:
Yeah keeping an eye on this issue to see if we need and can do something about it apart from having to create a 3rd package.